src/main/java/org/traccar/protocol/MeitrackProtocolDecoder.java

```diff
@@ -252,9 +255,15 @@ private Position decodeRegular(Channel channel, SocketAddress remoteAddress, Byt
                     value += (value < 0 ? -0.01 : 0.01) * Integer.parseInt(temp.substring(4), 16);
                     position.set(Position.PREFIX_TEMP + index, value);
                 }
+                index++;
             }
         }
-
+        if(parser.hasNext()){
+            position.set(Position.KEY_MAX_ACCELERATION,parser.nextInt());
+        }
+        if(parser.hasNext()){
+            position.set(Position.KEY_MAX_DECELERATION,parser.nextInt());
+        }
```

Comment:
Does this even compile? Looks like you are using non-existing keys. Also, you need to fix formatting